packages/flutter/lib/src/material/drawer.dart

```diff
@@ -26,6 +26,18 @@ const double _kEdgeDragWidth = 20.0;
 const double _kMinFlingVelocity = 365.0;
 const Duration _kBaseSettleDuration = const Duration(milliseconds: 246);
 
+/// A material design drawer.
+///
+/// Typically used in the [Scaffold.drawer] property, a drawer slides in from
+/// the side of the screen and displays a list of items that the user can
+/// interact with. The top-most item in a drawer is typically a [DrawerHeader]
+/// that displays status information about the current user.
```

Comment:
add a paragraph here suggesting putting the contents of the drawer in a Block to enable it to scroll?